vent/main.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python
 import argparse
 import sys
-from vent.coordinator.coordinator import get_coordinator
 from vent.gui.main import launch_gui
+from vent.coordinator.coordinator import get_coordinator
+
+from pudb.remote import set_trace
```

Comment:
Is this for debugging purposes?

Reply:
yes! it's a remote debugger that works with multiple processes, sorry a little debugging slop.